gapis/api/vulkan/framegraph.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vulkan
+
+import (
+	"context"
+
+	"github.com/google/gapid/core/log"
+	"github.com/google/gapid/gapis/api"
+	"github.com/google/gapid/gapis/service/path"
+)
+
+// GetFramegraph creates the framegraph of the given capture.
+func (API) GetFramegraph(ctx context.Context, p *path.Capture) (*api.Framegraph, error) {
+	log.W(ctx, "TODO: implement framegraph creation")
+	return &api.Framegraph{}, nil
```

Comment:
nit: I may choose returning nil instead of an empty struct.

Reply:
Done